packages/sign_in_with_apple/pubspec.lock

```diff
@@ -120,7 +162,7 @@ packages:
       name: test_api
       url: "https://pub.dartlang.org"
     source: hosted
-    version: "0.2.15"
+    version: "0.2.11"
```

Comment:
Also this gets downgraded on the AY user (Flutter 1.12), compared to my fresh dev account which is using `flutter dev` channel (as I need macOS support there).